tests/MPRThenRTOverlayNoHydration.spec.ts

```diff
@@ -10,11 +10,12 @@ test.beforeEach(async ({ page }) => {
 test('should launch MPR with unhydrated RTSTRUCT chosen from the data overlay menu', async ({
   page,
 }) => {
-  await page.getByTestId('side-panel-header-right').click();
-
   await page.getByTestId('Layout').click();
   await page.getByTestId('MPR').click();
 
+  // Wait 5 seconds for MPR to load. This is necessary in particular when screen shots are replace.
```

Comment:
I was wrong the sleeps are needed when adding/changing screen shots. I think we can change this going forward.